app/next-client-app/components/concepts/ConceptDetailsSheet.tsx

```diff
@@ -0,0 +1,381 @@
+                        ? values.confidence
+                        : parseFloat(values.confidence) || 0
+                      ).toFixed(2)}
+                    </div>
+                  </div>
+                </FormControl>
+                <FormMessage />
+              </FormItem>
+            )}
+          </FormField>
+          <div className="mt-4">
+            <FormField name="description">
+              {({ field }) => (
+                <FormItem>
+                  <FormLabel>Description</FormLabel>
+                  <FormDescription>
+                    Description of the concept mapping.
+                  </FormDescription>
+                  <FormControl>
+                    <Textarea
+                      {...field}
+                      placeholder="Enter description"
+                      onChange={handleChange}
+                      name="description"
+                      rows={3}
+                    />
+                  </FormControl>
+                  <FormMessage />
+                </FormItem>
+              )}
+            </FormField>
+          </div>
+          <div className="flex mt-4">
+            <Button type="submit" size="sm">
+              <Save className="mr-2 h-4 w-4" />
+              Save Changes
+            </Button>
+          </div>
+        </form>
+      )}
+    </Formik>
+  );
+}
+
+export function ConceptDetailsSheet({
+  concept,
+  children,
+  scanReportId,
+  tableId,
+  fieldId,
+  valueId,
+}: ConceptDetailsSheetProps) {
+  const [conceptDetail, setConceptDetail] =
+    useState<ScanReportConceptDetailV3 | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleUpdateConcept = (updatedDetail: ScanReportConceptDetailV3) => {
+    setConceptDetail(updatedDetail);
+  };
+
+  useEffect(() => {
+    if (isOpen && scanReportId && tableId && fieldId) {
+      const fetchConceptDetail = async () => {
+        setIsLoading(true);
+        try {
+          const detail = await getScanReportConceptDetail(
+            scanReportId,
+            tableId,
+            fieldId,
+            valueId?.toString() || "",
+            concept.id.toString()
+          );
+          setConceptDetail(detail);
+        } catch (error) {
+          console.error("Failed to fetch concept detail:", error);
+        } finally {
+          setIsLoading(false);
+        }
+      };
+
+      fetchConceptDetail();
+    }
+  }, [isOpen, scanReportId, tableId, fieldId, concept.id]);
+
+  return (
+    <Sheet open={isOpen} onOpenChange={setIsOpen}>
+      <SheetTrigger asChild>{children}</SheetTrigger>
+      <SheetContent className="flex flex-col h-full">
+        <SheetHeader className="flex-shrink-0">
+          <SheetTitle>
+            {concept.concept.concept_id} - {concept.concept.concept_name}
+          </SheetTitle>
+        </SheetHeader>
+        <div className="space-y-6 flex-1 overflow-y-auto">
```

Comment:
```suggestion
        <div className="px-1 space-y-6 flex-1 overflow-y-auto">
```
I think we need a padding here, otherwise the focus ring of the text input will be cut off.
If you want to make the title align with the rest, then you need to add `px-1` there as well, at `SheetHeader`

<img width="389" height="618" alt="Screenshot 2025-08-12 at 10 10 42" src="https://github.com/user-attachments/assets/a3bd9e9e-24e1-4023-bba4-ee9ae2b6aa08" />

Reply:
Thanks - yep have added